client.go

```diff
@@ -107,6 +131,42 @@ func (g *gethClient) Start(ctx context.Context, verbose bool) error {
 	return nil
 }
 
+// PostStart is called after the client has been fully started.
+// We send a forkchoiceUpdatedV2 request to the engine to trigger a post-merge forkchoice.
+func (g *gethClient) PostStart(ctx context.Context) error {
+	auth := node.NewJWTAuth(g.jwtauth)
+	endpoint := fmt.Sprintf("http://%s:%s", HOST, AUTHPORT)
+	cl, err := rpc.DialOptions(ctx, endpoint, rpc.WithHTTPAuth(auth))
+	if err != nil {
+		return err
+	}
+	defer cl.Close()
+
+	geth := ethclient.NewClient(cl)
+
+	block, err := geth.BlockByNumber(ctx, nil)
+	if err != nil {
+		return err
+	}
+
+	var (
+		tstamp       = uint64(time.Now().Unix())
+		feeRecipient = common.HexToAddress("fee")
+		fcResponse   engine.ForkChoiceResponse
+	)
+	fmt.Println("called forkchoiceUpdatedV2 with block hash:", block.Hash().String(), "and timestamp:", tstamp, "and feeRecipient:", feeRecipient.String())
+	err = cl.CallContext(ctx, &fcResponse, "engine_forkchoiceUpdatedV2", &engine.ForkchoiceStateV1{
+		HeadBlockHash:      block.Hash(),
```

Comment:
@lightclient I think after this engine request, we can query the `safe,fianalized` tags(the result is the same as `latest`).